xds/internal/server/listener_wrapper.go

```diff
@@ -297,11 +302,22 @@ func (l *listenerWrapper) Accept() (net.Conn, error) {
 			conn.Close()
 			continue
 		}
-		// TODO: once matched an accepted connection to a filter chain,
-		// instantiate the HTTP filters in the filter chain + the filter
-		// overrides, pipe filters and route into connection, which will
-		// eventually be passed to xdsUnary/Stream interceptors.
-		return &connWrapper{Conn: conn, filterChain: fc, parent: l}, nil
+		var rc xdsclient.RouteConfigUpdate
+		if fc.InlineRouteConfig != nil {
+			rc = *fc.InlineRouteConfig
+		} else {
+			rcPtr := l.rdsUpdates
+			rcuPtr := (*map[string]xdsclient.RouteConfigUpdate)(rcPtr)
```

Comment:
What is `rcuPtr` is `nil`? I guess that's not a case we expect. But I think it would still be better to handle that with some kind of error. Maybe return an error from `Accept`, and that will case the server to be shutdown. Or just log and continue? We definitely don't want a panic.

Reply:
Why make this `*map` and not `map`?  Does `map` not work?  Maps are already pointers in Go.